1st try (3rd lab)

M2203114 opened the pull request.

M2203114 committed:
1st try (3rd lab)

NemirovichMark commented:
Hello Egor.

Optimizing (for the future):
while (true)
{
    if (!double.TryParse(Console.ReadLine(), out double x))
    {
        Console.WriteLine("Incorrect value");
        continue;
    }
    arr.Add(x);
    break;
}

can be shorted
while (!double.TryParse(Console.ReadLine(), out x))
{
    Console.WriteLine("Incorrect value");
}
arr.Add(x);

1 lvl Ex11() Mistake
The task is: from input array's elements move positive to new array. You do not allow to create input array with negative elements at all.

2 lvl Ex13()  Too consuming solution. No need to use List here. Use simple array, please. 

2 lvl Ex15() It is inserting before the element. According to the task: after k-element (in the example: after 2)
![image](https://user-images.githubusercontent.com/112826355/198041889-08a0dd86-8a25-413e-93da-a303cfe4d1e6.png)

Algo 12: It have to work with different arrays (n) and (m). Read the last paragraph of the task.


Good luck!

M2203114 committed:
3rd lab (second attempt)

NemirovichMark commented:
Hello Egor.

Algo 12. Did you check your program after writting?
![image](https://user-images.githubusercontent.com/112826355/198248916-e81ff713-e786-45bd-bd90-6cd1f3320b40.png)

M2203114 committed:
3rd one

To be honest, no. I thought that nothing can happen after minor changes. From now on I'll be more attentive

NemirovichMark commented:
Good job Egor!
